index.html

```diff
@@ -282,6 +282,7 @@ <h4>LET'S CONNECT</h4>
       </section>
 
       <section class="contact-me">
+        <br />
         <h6>Contact me</h6>
         <p>
           If you have an application you are interested in developing, a feature

```

Comment:
### OPTIONAL

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_181](https://user-images.githubusercontent.com/69638013/168232983-da16558b-93b0-4734-aa88-929e25c4b5cc.png)  | ![Screenshot_180](https://user-images.githubusercontent.com/69638013/168232950-4b178cc5-46e1-4812-b4ca-51b092fcca6d.png)

- [ ] Good job making all the requested changes 👍🏿  Kindly increase the with of the paragraph text to have the same width as the form
- [ ] Please change the  color for name and email inputs to the color used in the Figma design 👍🏿